Source/SlimDetours/Transaction.c

```diff
@@ -54,6 +54,18 @@ static PDETOUR_OPERATION s_pPendingOperations = NULL;
 HRESULT
 NTAPI
 SlimDetoursTransactionBegin(VOID)
+{
+    DETOUR_TRANSACTION_OPTIONS Options;
+    Options.cbSize = sizeof(Options);
+    Options.fSuspendThreads = TRUE;
+
+    return SlimDetoursTransactionBeginEx(&Options);
+}
+

```

Comment:
This can be moved to public header `SlimDetours.h` after new added `SlimDetoursTransactionBeginEx`:
```C
FORCEINLINE
HRESULT
SlimDetoursTransactionBegin(VOID)
{
    ...
}
```